downloadable/playercards/Unofficial Return to Night of the Zealot  Investigator Expansion.json

```diff
@@ -0,0 +1,10048 @@
+{
+  "SaveName": "",
+  "Date": "",
+  "VersionNumber": "",
+  "GameMode": "",
+  "GameType": "",
+  "GameComplexity": "",
+  "Tags": [],
+  "Gravity": 0.5,
+  "PlayArea": 0.5,
+  "Table": "",
+  "Sky": "",
+  "Note": "",
+  "TabStates": {},
+  "LuaScript": "",
+  "LuaScriptState": "",
+  "XmlUI": "",
+  "ObjectStates": [
```

Comment:
This outer layer needs to be removed (TTS adds this to the JSON data of an object to make it a "pseudo-savegame") - we only want what's inside the `ObjectStates` field.

So lines 1-18 and the last two (that close the open parenthesis) need to get removed.